src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.kfs.fp.document.web.struts;
+
+import java.io.Serializable;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.struts.action.ActionForm;
+import org.apache.struts.action.ActionForward;
+import org.apache.struts.action.ActionMapping;
+import org.kuali.kfs.sys.KFSKeyConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.krad.service.DocumentService;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.cornell.kfs.fp.service.RecurringDisbursementVoucherDocumentService;
+
+public class RecurringDisbursementVoucherAction extends CuDisbursementVoucherAction implements Serializable {
+
+    private static final long serialVersionUID = -6417266354906825200L;
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(RecurringDisbursementVoucherAction.class);
+    private static final String DOT = ".";
```

Comment:
It doesn't look like our version of KFSConstants has a DELIMITER value.  I added one to CUKFSConstants.